.github/workflows/delete-unused-step-functions.yaml

```diff
@@ -29,39 +29,49 @@ jobs:
       - id: remove-old-state-machines
         name: Remove old step functions
         run: |-
-          threshold=500
```

Comment:
@cosa65 can you also remove `HMS` from the `environment-name` list? I realised that this workflow runs only for staging and we don't need to run this for staging for HMS. Or alternatively, add these lines in the yaml file after `environment-name`:

```
        exclude:
          - environment-name: HMS
            environment-type: staging
```
We already do this in other workflows (see deploy-infra.yaml)

Reply:
talked over slack, it only runs once for each environment-name